### PR TITLE
[TASK] Document default language configuration for user interface translation

### PR DESCRIPTION
**Pull request by testbird**

### CODE REVIEW

File: TYPO3.Neos/Documentation/InsideOfNeos/UserInterface/Translation.rst [hunk 1]
```diff
@@ -2,10 +2,24 @@
...
 
+Default Language
+================
+The `availableLanguages` are defined in `Packages/Application/TYPO3.Neos/Configuration/Settings.yaml`.
+
+You may override the default language of your installation in `Configuration/Settings.yaml`:
+
+.. code-block:: yaml
+
+  TYPO3
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/InsideOfNeos/UserInterface/Translation.rst]: missing colon :

File: TYPO3.Neos/Documentation/InsideOfNeos/UserInterface/Translation.rst [hunk 2]
```diff
@@ -2,10 +2,24 @@
 Translating the user interface
 ==============================
 
+Default Language
+================
+The `availableLanguages` are defined in `Packages/Application/TYPO3.Neos/Configuration/Settings.yaml`.
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/InsideOfNeos/UserInterface/Translation.rst]: needs to be two ` marks to work, see https://github.com/testbird/neos-development-collection/blob/patch-1/TYPO3.Neos/Documentation/InsideOfNeos/UserInterface/Translation.rst

File: TYPO3.Neos/Documentation/InsideOfNeos/UserInterface/Translation.rst [hunk 3]
```diff
@@ -2,10 +2,24 @@
...
+
+You may override the default language of your installation in `Configuration/Settings.yaml`:
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/InsideOfNeos/UserInterface/Translation.rst]: same here

**Comment by aertmann** [comment]
@testbird: Hey great to see a push from you. Thanks a lot. Looks good already, but left a couple of nit pick comments. Please adjust the code and update pull request, then we can merge it.

**Comment by testbird** [comment]
Ok, done. But can't find the button to update the pull request.

**Comment by dimaip** [comment]
Hey @testbird, the PR is updated automatically. Great to see you up to speed with contributing!

**Comment by aertmann** [comment]
:+1:

**Comment by kdambekalns** [comment]
Thanks!

**Pull request merged**